client/src/pages/Plan/common/Panels/data.jsx

```diff
@@ -76,6 +77,12 @@ export const DataPanel = ({
 						</Moment>
 					</li>
 				</SC.Datalist>
+				{originalName && (
```

Comment:
is this outside of the Datalist component on purpose? (unlike all above list items)

Reply:
fixed